### PR TITLE
Embed Freshdesk contact form into support docs and reword a little

### PR DESCRIPTION
**Pull request by sgibson91**
fixes https://github.com/2i2c-org/meta/issues/452

### CODE REVIEW

**Comment by sgibson91** [comment]
So the widget isn't showing in the build preview: https://2i2c-pilot-documentation--180.org.readthedocs.build/en/180/support.html

@choldgraf Do I have to do anything special to get sphinx and html working together?

**Comment by sgibson91** [comment]
Maybe I have to define the button? I thought that would have come with the embed code though https://support.freshdesk.com/en/support/solutions/articles/50000001015-launching-the-widget-when-a-button-is-clicked

**Comment by choldgraf** [comment]
HTML should work, but I wonder if you need to have a space between it and the comment?

File: support.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
-Send all support requests as an email to [**`support@2i2c.org`**](mailto:support@2i2c.org).
-This will be routed to the 2i2c support team, and we will get back to you shortly!
+Send all support requests using the contact form widget on the bottom left of this page.
+Click the "Help" button to expand it.
+Requests submitted via this form will be routed to the 2i2c support team, and we will get back to you shortly!
 When you make a support request, please include as much information as possible in order to provide context needed to resolve your issue!
 
-% hard-coding this because sphinx-design buttons don't work with mailto
-<a class="sd-sphinx-override sd-btn sd-text-wrap sd-btn-primary reference external" href="mailto:support@2i2c.org"><span>Send a support email</span></a>
+% Embed a contact form Freshdesk widget to collect support tickets with
```
Comment (sgibson91) [inline support.md:9]:
```suggestion
% Embed a contact form Freshdesk widget to collect support tickets with

```

**Comment by sgibson91** [comment]
Hmmm, I don't think that worked :-/

**Comment by choldgraf** [comment]
It looks like it worked to me - the docs you linked suggest that the button exists in the bottom right of the screen, and I do see that in the preview:

image

The first step of the docs you linked also suggest we need to create our own button if we want people to click something to open the menu _other than_ clicking the hover button in the bottom right:

image

I added a little HTML button snippet following the docs they recommended in that article and pushed in the latest commit. It should work if you click it, but people should also be able to click the little "help icon" in the bottom right too.

### Question

Should we put that help icon in the bottom right on _every page_ rather than just the support page?

**Comment by sgibson91** [comment]
Clicking the HTML button you added also does not open anything for me, so I'm guessing there's something not compatible about my browser

**Comment by sgibson91** [comment]
Ok, PrivacyBadger was blocking the pop-up on my browser. So we may need to add a warning for people to check their browser privacy settings if they don't see the Help button?

**Comment by sgibson91** [comment]
> Should we put that help icon in the bottom right on _every page_ rather than just the support page?

I don't think this is a terrible idea BUT I'm not sure how to implement it without copy-pasting that snippet onto every single page, and remembering to add it to every new page we add. So I think that is a stretch goal we should reserve for later on in the development.

**Comment by choldgraf** [comment]
Want me to give it a shot quickly? I think it should be doable with a little sphinx-foo

**Comment by sgibson91** [comment]
@choldgraf Sure, if you think you can do it! I am lacking on sphinx-foo 😄

**Comment by choldgraf** [comment]
OK I think the last commit adds the JS to each page. I also tried to document things so it's clear where they come from

**Comment by sgibson91** [comment]
Amazing! It works for me!

**Comment by sgibson91** [comment]
I will have lunch and then add a note about browser privacy settings, then I will mark this PR as ready to review

**Review by damianavila — APPROVED** [review]
LGTM.

**Pull request merged**